http/hibernate-validator/src/test/java/io/quarkus/ts/http/hibernate/validator/BaseResteasyIT.java

```diff
@@ -22,33 +22,64 @@ public abstract class BaseResteasyIT {
 
     private static final String EXPECTED_VALIDATOR_ERROR_MESSAGE = "numeric value out of bounds";
 
-    protected void assertBadRequestInJsonFormat(String path) {
-        assertBadRequestInJsonFormat(given().get(path));
+    protected static ResponseValidator validate(String path) {
+        return BaseResteasyIT.validate(given().get(path));
     }
 
-    protected void assertBadRequestInJsonFormat(Response response) {
-        isBadRequest(response)
-                .contentType(ContentType.JSON)
-                .body("parameterViolations[0].message", containsString(EXPECTED_VALIDATOR_ERROR_MESSAGE));
+    protected static ResponseValidator validate(Response response) {
```

Comment:
If this is a methodFactory of `ResponseValidator` I would add a static method to ResponseValidator in order to create instances. 

```
    protected static ResponseValidator validate(String path) {
        return ResponseValidator.getInstance(given().get(path));
    }

//    protected static ResponseValidator validate(Response response) {
//        return new ResponseValidator(response);
//    }

    static class ResponseValidator {
        final Response response;
        
        public static ResponseValidator getInstance(Response response) {
            return new ResponseValidator(response);
        }

        private ResponseValidator(Response response) {
            this.response = response;
        }
```
Or implement a builder ...

Reply:
Well, is there any difference, except for making +1 word at the call place?

Reply:
But you have a method `validate` that creates instances of ResponseValidator... which actually does the validation. Mmm... to me sound weird in terms of "responsibility" (design) . I would go for a private ResponseValidator constructor(so by this way you are sure that the `validate` factoryMethod is used), and also I would move the method to `ResponseValidator` class. Because make sense that the method that is used to create instances of a class is "part" of this class...Is a static method so...is not really part of the class but well...at least is easy to find, making sense from a conceptual point of view.  
And also, the name `validate` doesn't reflect what this method actually is doing. 

WDYT? @fedinskiy

Reply:
After discussion in private it turned out, that this class should not have both public constructor and outside building method. Fixed